src/DataFixtures/ClientFixtures.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+use App\Entity\Client;
+use Exception;
+
+/**
+ * Class ClientFixtures
+ *
+ * @package App\DataFixtures
+ */
+class ClientFixtures extends Fixture
+{
+    public const CLIENT_REFERENCE = 'client';
+
+    /**
+     * @param ObjectManager $manager
+     *
+     * @throws Exception
+     */
+    public function load(ObjectManager $manager)
+    {
+        for ($i = 1; $i <= 10; $i++) {
+            $client = new Client();
+            $client->setEmail('email' . $i++ . '@email.fr');
+            $client->setPassword('password');
+            $manager->persist($client);
+        }
+
+        $manager->flush();
+
+        $this->setReference(self::CLIENT_REFERENCE, $client);
```

Comment:
Remonter la référence dans la boucle, car le but est de créer autant de référence client qu'il y a de client, exemple :
`$this->setReference(self::CLIENT_REFERENCE . "_" . $i, $client);`

Reply:
J'ai modifié les fixtures voir le commit bffc6befbe9ab0ff0da90782c56e4c45eb5decf8.